src/components/projects/EditorElement.tsx

```diff
@@ -119,7 +120,11 @@ const EditorElement: React.FC = () => {
       <SpriteMap color="#353535" />
       {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions  */}
       <div id="editorDropzone" ref={drop}>
-        <div id="editor" ref={onRender}></div>
+        <div
+          id="editor"
+          key={`editor-mode-${isViewingMode ? 'view' : 'edit'}`}
```

Comment:
are we doing this to force react recreate the html? Can you please add a comment in the code what it's for.

Reply:
Correct. Please let me know if the added comment makes the purpose clear.

Reply:
Looks great. Thanks!